### PR TITLE
docs: add weekly report 0716-0731

### PR DESCRIPTION
**Pull request by lijing-21**
add weekly report 0716-0731

### CODE REVIEW

File: blog/en/blog/2022/08/09/weekly-report-0731.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Apache APISIX Ingress Controller v1.5-rc1 is officially released. Bring API Version upgrades, Gateway API support, and updates to Ingress resources.
+
+- [Release Apache APISIX 2.15](https://apisix.apache.org/blog/2022/07/29/release-apache-apisix-2.15/)
+
+Apache APISIX 2.15 is officially released! You can customize plugin priority and whether the plugin is executed, custom error response, and indicators to support the monitoring of four layers of traffic.
+
+- [Why do you need Apache APISIX when you have NGINX and Kong?](https://apisix.apache.org/blog/2022/07/30/why-we-need-apache-apisix/)
+
+This article describes the history of the open source API Gateway Apache APISIX architecture's evolution and compares the advantages of the two frameworks, Apache APISIX and Nginx.
```
Comment (SylviaBABY) [inline blog/en/blog/2022/08/09/weekly-report-0731.md]:
```suggestion

This article describes the history of the open source API Gateway Apache APISIX architecture's evolution and compares the advantages of the two frameworks, Apache APISIX and NGINX.
```

File: blog/en/blog/2022/08/09/weekly-report-0731.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- [Add plugin_metadata into control API](https://github.com/apache/apisix/pull/7514)（Contributor: [kingluo](https://github.com/kingluo)）
+
+- [Support multi clickhouse endpoints in `clickhouse-logger` plugin](https://github.com/apache/apisix/pull/7517)（Contributor: [zhendongcmss](https://github.com/zhendongcmss)）
+
+## Recent Blog Recommendations
+
+- [Release Apache APISIX Ingress v1.5-rc1](https://apisix.apache.org/blog/2022/08/05/apisix-ingress-1.5rc1-release/)
+
+Apache APISIX Ingress Controller v1.5-rc1 is officially released. Bring API Version upgrades, Gateway API support, and updates to Ingress resources.
```
Comment (SylviaBABY) [inline blog/en/blog/2022/08/09/weekly-report-0731.md]: image
the same as CN version

File: blog/en/blog/2022/08/09/weekly-report-0731.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+> From Jul 16th to Jul 31th, 28 contributors submitted 93 commits for Apache APISIX. Thank you all for your contributions to Apache APISIX. It is your selfless contribution to make the Apache APISIX project better!
+
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX has grown as a community from the first day of open source and has quickly become the most active open source API gateway project in the world. These achievements cannot be achieved without the joint efforts of our community partners.
+
+"If you want to go fast, go alone.If you want to go far, go together." The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
```
Comment (guitu168) [inline blog/en/blog/2022/08/09/weekly-report-0731.md]:
```suggestion
"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and facilitate everyone to participate in the Apache APISIX community.
```

**Comment by SylviaBABY** [comment]
the cover is not correct
image

**Comment by guitu168** [comment]
> the cover is not correct image

@SkyeYoung  has fixed this issue. cc @SylviaBABY

**Review by SylviaBABY — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**